.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug"]
```

Comment:
I am not sure I like labeling an issue as a bug when a bug report is filed. Now the "bug" label is assigned manually by a maintainer and I assign them only when I know the issue isn't a user error and someone was able to reproduce it. That said, I can see the other point of view - all bug reports should be labeled as such.

Reply:
The form can apply more labels, which could be `unconfirmed` or `needs-triage`, to be removed by a maintainer when confirmed.

Another approach is keeping the automatic labels as-is, but adding to the repo the usage of a `confirmed` label, to be added by a maintainer when the bug in confirmed.  
This might be better if there's interest in excluding confirmed bugs from stalebot.

The idea of both approaches is to ensure that anything claimed to be a bug can actually be found without fumbling with github's horrible keyword search. Searching by labels is more capable and allows searching for combinations, ie: `label:bug -label:unconfirmed` to only find confirmed bugs.

Reply:
I like the idea of an "unconfirmed" label :+1:

Reply:
if you create the label, we can add it to the PR

Reply:
Any news on this?

Reply:
added the new `needs-testing` label to the bug report template by default, which should solve this.